tap-gui/troubleshooting.md

```diff
@@ -238,4 +237,30 @@ app_config:
     reading:
       allow:
       - host: acc-server.accelerator-system.svc.cluster.local
-```
\ No newline at end of file
+```
+
+## <a id='source-scan-no-cves'></a> Source Scan stage may not display CVEs
+
+### Symptom
+
+Source Scan stage vulnerabilities table displays "No records to display" even though there are CVEs in the source scan.
+
+![Screenshot of source scan stage not showing any CVEs](images/troubleshooting-no-cves-source-scan.png)
+
+### Solution
+
+If no CVEs are displayed, run `kubectl describe sourcescan <workload name> -n <workload namespace>` to see if any CVEs were found during the source scan. If so, use the tanzu insight CLI plug-in to query for vulnerability details.  See [Query vulnerabilities, images, and packages](../cli-plugins/insight/query-data.md).
+
+
+## <a id='maven-artifact-error'></a> Maven artifacts access error
+
+### Symptom
+
+Source Scan stage vulnerabilities table displays "No records to display" even though there are CVEs in the source scan.
+
+![Screenshot warning of no Maven artifact access ](../images/../tap-gui/images/tap-gui-maven-artifact-1.png)
+
+### Solution
+
+If no CVEs are displayed, run `kubectl describe sourcescan <workload name> -n <workload namespace>` to see if any CVEs were found during the source scan. If so, use the tanzu insight CLI plug-in to query for vulnerability details.  See [Query vulnerabilities, images, and packages](../cli-plugins/insight/query-data.md).
```

Comment:
The screenshot has been swapped out but not the instructions

Reply:
@mstergianis , thank you for making these changes! Are we good to merge?

Reply:
Uhh, I haven't made any changes. But I can

Reply:
@mstergianis, are you in the midst of making the requested changes (do you have a local feature branch or similar?) I don't want to merge this prematurely.

Reply:
Hey sorry @RichardJJG this one fell through the cracks as I got distracted by other things. I'll take a look at throwing something together today. But it'll be a separate PR. I'm making an assumption here, but I don't think I have push access to Andrei's fork.

Reply:
Perfectly fine to make a separate PR, thanks, @mstergianis